modules/apps/style-book/style-book-api/src/main/resources/com/liferay/style/book/service/packageinfo

```diff
@@ -1 +1 @@
-version 5.0.0
```

Comment:
Usually we create a "semantic version" commit  for `packageinfo` and `bnd` files version changes only :slightly_smiling_face:

Reply:
@algobob Thanks for the review! ❤️ 
Should I split the auto-generated code by buildService into two distinct commits?
Thanks!

Reply:
Please @lfbesada, merge these changes with the `buildService` commit. They are auto-generated by buildService.